resources/ko-KR/spaceResorts.hints.bxb

```diff
@@ -0,0 +1,7 @@
+hints {
+  uncategorized {
+    utterance ("[g:SpaceResort#all] (금성)[v:Planet:Venus]에 (무중력 골프장)[v:SearchCriteria]이 있는 리조트 보여줘봐")
+    utterance ("[g:SpaceResort#all] (해왕성)[v:Planet:Neptune]에 (키즈룸)[v:SearchCriteria]이 있는 호텔 찾아봐")
+    utterance ("[g:SpaceResort#all] (토성)[v:Planet:Saturn] 근처 리조트 검색해줘")
```

Comment:
Recommendation would be to use plain NL instead of aligned-nl

Reply:
Ok, I have changed it